Wrangler2 release (#3909)
cloudflare/cloudflare-docs: content/workers/wrangler/custom-builds.md

---
pcx-content-type: how-to
title: Custom Builds
weight: 4
---

> 

Reviewer: This needs a "WORK IN PROGRESS" banner.

## Custom builds

> While `wrangler` provides a capable development environment for a large number of use cases, developers often face the need to customise how their code is compiled before being processed by Wrangler. This document suggests some strategies for orchestrating custom build commands and systems in conjunction with `wrangler dev` and `wrangler publish`.

Reviewer 2: TODO!

In particular it would be good to cover some of the use cases where we used to need webpack.

> While `wrangler` provides a capable development environment for a large number of use cases, developers often face the need to customise how their code is compiled before being processed by Wrangler. This document suggests some strategies for orchestrating custom build commands and systems in conjunction with `wrangler dev` and `wrangler publish`.

Reviewer 3: ```suggestion
While `wrangler` provides a capable development environment for a large number of use cases, developers often need to customize how their code is compiled before being processed by Wrangler. This document suggests some strategies for creating custom build commands and systems together with `wrangler dev` and `wrangler publish`.
```
